src/miniscript/mod.rs

```diff
@@ -128,6 +128,8 @@ impl<Pk: MiniscriptKey> Miniscript<Pk> {
 impl Miniscript<bitcoin::PublicKey> {
     /// Attempt to parse a script into a Miniscript representation
     pub fn parse(script: &script::Script) -> Result<Miniscript<bitcoin::PublicKey>, Error> {
+        // Transactions more than 100Kb are non-standard
+        if script.len() > 10000 {}
```

Comment:
Is this missing to return Err(ScriptSizeTooLarge) ?
Is it missing a 0 to be 100k?

Reply:
Thanks for the reivew :) . Good catch, will fixup shortly